directanswercards/documentsearch-standard/component.js

```diff
@@ -21,11 +21,16 @@ class documentsearch_standardComponent extends BaseDirectAnswerCard['documentsea
       snippetValue = Formatter.highlightField(snippet.value, snippet.matchedSubstrings);
     }
 
+    let viewDetailsUrl = relatedItemData.website || (relatedItemData.fieldValues && relatedItemData.fieldValues.landingPageUrl);
```

Comment:
maybe this whole thing should go into the formatter? I think we definitely want the isChrome part to go into the formatter at least, and if the hitchhikers never need to touch stuff like relatedItemData.website maybe that can go in their too (maybe Rose would have a better opinion here)

Reply:
hm good point, I will update this after she respond

Reply:
Updated! we will pass the url as a param since this might be apply in other areas in the future. I moved isChrome to the formatter.